test/Project.toml

```diff
@@ -16,7 +16,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 CSV = "0.10"
-CairoMakie = "0.13"
+CairoMakie = "0.12"
```

Comment:
So, uhm, SciMLBase is only compatible with CairoMakie up to 0.12. No idea why this worked before and now stopped working.
My best guess is that the compat was only recently added to SciMLBase, and we were not using the latest version of SciMLBase before.

Reply:
https://github.com/SciML/SciMLBase.jl/issues/909